### ISSUE TITLE
TPU MNIST example: incorrect normalization of image values

### ISSUE THREAD

**Issue by bzamecnik** [tensorflow/tpu#435]
In https://github.com/tensorflow/tpu/blob/master/tools/colab/keras_mnist_tpu.ipynb:

```
def read_image(tf_bytestring):
    image = tf.decode_raw(tf_bytestring, tf.uint8)
    image = tf.cast(image, tf.float32)/256.0
    image = tf.reshape(image, [28*28])
    return image
```

should be:

```
    image = tf.cast(image, tf.float32)/255.0
```

The reasons: in float representation the image range is [0.0, 1.0] and in int it is [0, 255]. In order to map 255 to 1.0 the normalization factor needs to be 255, not 256.

Also make sure the same factor is used in the opposite operation while preparing the dataset.
